source/test/nebula/ecs/system/DrawSpriteSystemTest.hx

```diff
@@ -62,5 +68,32 @@ class DrawSpriteSystemTest
         Assert.that(sprite.x, Is.equalTo(p.x));
         Assert.that(sprite.y, Is.equalTo(p.y));
     }
+
+    @Test
+    public function isReaptingConvertsTheSpriteSpriteToFlxBackdrop()
```

Comment:
I think you spelled "repeating" wrong (typo). I like to name tests based on the function under test. Do you mind renaming this to "updateInitializesRepeatingSpriteToFlxBackdrop"?